deps.toml

```diff
@@ -11,7 +11,7 @@ jmh = "1.36"
 jooq = "3.13.4"
 junit-jupiter = "5.9.1"
 kotlin = "1.9.0"
-log4j = "2.17.2"
+log4j = "2.21.1"
```

Comment:
:tada:

I think this will also fix some weird issues I've been having in intellij with tests not logging correctly!

Reply:
It's possible that that's because we have two competing log4j configs on the classpath.